.github/actions/update-csv-bundles/action.yml

```
@@ -0,0 +1,173 @@
+
+        echo "Setting version in application.yaml"
+        sed -i \
+          "s/    version: \"[0-9]+\.[0-9]+\.[0-9]+\"/    version: \"${VERSION}\"/" \
+          "config/helm/chart/default/templates/application.yaml"
+    - name: Install operator-sdk v1.36.0
+      shell: bash
+      env:
+        VERSION: ${{ inputs.version }}
+      run: |
+        GOBIN="$(go env GOPATH)/bin"
+        mkdir -p "$GOBIN"
+
+        echo "Installing operator-sdk v1.36.0 into $GOBIN"
+        curl -OJL \
+          "https://github.com/operator-framework/operator-sdk/releases/download/v1.36.0/operator-sdk_linux_amd64"
+        chmod +x operator-sdk_linux_amd64
+
+        mv operator-sdk_linux_amd64  ${GOBIN}/operator-sdk
+        "${GOBIN}/operator-sdk" version
+
+        echo "GOBIN=$GOBIN" >> $GITHUB_ENV
+    - name: Generate OLM bundles (openshift)
+      shell: bash
+      env:
+        VERSION: ${{ inputs.version }}
+      run: |
+        VERSION="${VERSION}"
+        mkdir -p config/olm/openshift/bases
+        make bundle PLATFORM=openshift VERSION="${VERSION}" CHANNELS=stable DEFAULT_CHANNEL=stable OLM_IMAGE="registry.connect.redhat.com/dynatrace/dynatrace-operator:v${VERSION}" IMAGE="registry.connect.redhat.com/dynatrace/dynatrace-operator"
+    - name: Generate OLM bundles (kubernetes)
+      shell: bash
+      env:
+        VERSION: ${{ inputs.version }}
+      run: |
+        VERSION="${VERSION}"
+        make bundle PLATFORM=kubernetes VERSION="${VERSION}" CHANNELS=stable DEFAULT_CHANNEL=stable IMAGE="docker.io/dynatrace/dynatrace-operator" OLM_IMAGE="docker.io/dynatrace/dynatrace-operator:v${VERSION}"
+    - name: Finalize CSV files
+      shell: bash
+      env:
+        VERSION: ${{ inputs.version }}
+      run: |
+        VERSION="${VERSION}"
+
+        echo "Finalizing CSV files for Kubernetes"
+        ./.github/scripts/finalize_csv_files.py \
+          --platform "kubernetes" \
+          --version "${VERSION}" \
+          --changelog "${{ inputs.changelog-path }}" \
+          --token "${GITHUB_TOKEN}"
+
+        echo "Finalizing CSV files for Openshift"
+        ./.github/scripts/finalize_csv_files.py \
+          --platform "openshift" \
+          --version "${VERSION}" \
+          --changelog "${{ inputs.changelog-path }}" \
+          --token "${GITHUB_TOKEN}"
+
+    - name: Create CSV update PR
+      uses: peter-evans/create-pull-request@v7
+      env:
+        VERSION: ${{ inputs.version }}
+      with:
+        base: release-${{ steps.vars.outputs.major_minor }}
+        branch: csv-${{ steps.vars.outputs.major_minor}}
+        delete-branch: true
+        add-paths: |
+          config/olm/**
+          config/deploy/kubernetes/kustomization.yaml
+          config/deploy/openshift/kustomization.yaml
+          config/manifests/bases/**
+          config/helm/**
+        commit-message: "chore: update OLM bundles for v${{ env.VERSION }}"
+        signoff: true
+        title: "Update OLM CSV bundles for v${{ env.VERSION }}"
```

```suggestion
        title: "Update OLM CSV bundles for v${{ inputs.version }}"
```
i think we can use it directly but please double check it.